### PR TITLE
CMS prep - Update redirects batch 1

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates redirected files in preparation for the move to a CMS.

## Links
Jira ticket: [ACQ-2827](https://codedotorg.atlassian.net/browse/ACQ-2827)
Gsheet: [here](https://docs.google.com/spreadsheets/d/1Wwes9eF375MRNyGAx83KpVspbmDGbDsU-bBMwUV-WCc/edit)

## Testing story
Tested locally to make sure each redirect works, and made sure the number of updated files on the gsheet match the "Files changed" in this PR.

Screenshot 2024-11-22 at 1 08 16 PM

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/csp/unit1.moved [hunk 1]
```diff
@@ -1 +1 @@
-https://curriculum.code.org/csp/unit1
```
Comment (bencodeorg) [inline pegasus/sites.v3/code.org/public/educate/csp/unit1.moved:1]: Any reason not to make these relative paths?

**Review by bencodeorg — APPROVED** [review]
🎉

File: pegasus/sites.v3/code.org/public/educate/csp/unit1.moved (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/educate/csp/unit1.moved:1]: The relative path sends you [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites/virtual/curriculum-csp/index.haml), which is weirdly in the `pegasus/sites` directory, not `sites.v3` where everything else is. I didn't want to chance that going live so I used the hardcoded url.

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/educate/csp/unit1.moved:1]: I don't think it's worth figuring this out at this point, but if you/others disagree I can make a ticket to follow up!

Comment (bencodeorg) [reply pegasus/sites.v3/code.org/public/educate/csp/unit1.moved:1]: Interesting, good catch! Yeah agreed, no objection to leaving it a mystery for now :)

**Pull request merged**